whats_left.sh

```diff
@@ -8,4 +8,4 @@ python3 not_impl_gen.py
 
 cd ..
 
-cargo run -- tests/snippets/whats_left_to_implement.py
+cargo +nightly run -- tests/snippets/whats_left_to_implement.py
```

Comment:
Please leave this file as it is. Other people also depend upon this one.

Reply:
Ok I wasn't intented to push this change but I forgot that I changed it. I will revert it back with my next commit.